tracing-example/pom.xml

```diff
@@ -38,8 +38,8 @@
         <version.org.jboss.resteasy.extensions>2.0.1.Final</version.org.jboss.resteasy.extensions>
         <version.org.junit>5.10.1</version.org.junit>
         <version.org.wildfly.arquillian>5.0.1.Final</version.org.wildfly.arquillian>
-        <version.wildfly-maven-plugin>4.2.1.Final</version.wildfly-maven-plugin>
-        <version.org.wildfly>28.0.0.Beta1</version.org.wildfly>
+        <version.wildfly-maven-plugin>5.0.0.Alpha2</version.wildfly-maven-plugin>
```

Comment:
I'm trying to decide how I feel about using an Alpha release in an example. Examples should be able to be modified to use in production. However, by adding a Alpha dependency makes it not feel like production quality.

That said, this is a plugin version so maybe it doesn't matter as much. It might though :)